ui/app/mainui/AppMain.qml

```diff
@@ -762,6 +768,20 @@ Item {
             }
         }
 
+        Component {
+            id: activityCenterPopupComponent
+            ActivityCenterPopup {
+                id: activityCenter
+                height: appView.height - 56 * 2 // TODO get screen size // Taken from old code top bar height was fixed there to 56
```

Comment:
Time to fix this now that we are in AppMain? ;)

Reply:
okay, agreed :D